src/command/cmd_funcs.c

```diff
@@ -8580,31 +8579,24 @@ _update_presence(const resource_presence_t resource_presence,
 }
 
 // helper function for boolean preference commands
-static void
+static gboolean
 _cmd_set_boolean_preference(gchar* arg, const char* const command,
                             const char* const display, preference_t pref)
 {
     if (arg == NULL) {
         cons_bad_cmd_usage(command);
-    } else if (strcmp(arg, "on") == 0) {
-        GString* enabled = g_string_new(display);
-        g_string_append(enabled, " enabled.");
-
-        cons_show(enabled->str);
+        return FALSE;
+    } else if (g_strcasecmp(arg, "on") == 0) {
```

Comment:
Usually we use case sensitive comparison. This one is fine for me as well. Just thinking whether it might be strange if it works in some cases case sensitive and in others not.

Reply:
> Usually we use case sensitive comparison

why? I can understand case sensitivity for some things, but ON vs. On vs. on vs. oN shouldn't make a difference ... this is als true for other settings, but that's the first one where I've changed it.

IMO Postel's law is very important when it comes to usability of software.

Reply:
I'm not saying we SHOULD use case sensitive comparison.
I'm only saying that so far we do. And I would find it weird to now change this in only *one* occasion :)
I think it should be uniform.

From the top of my head I don't remember whether it actually is important/needed that we do that. So we could change it everywhere to be case insensitive. But I don't actually see any advantage it that. It might be that we miss something where it would make sense to be case sensitive. And then we have to change everything back. Plus it would just create work.

Those to reasons are why I think it it should be changed. The main reason is to be uniform though.

And I also see:
```
g_strcasecmp has been deprecated since version 2.2 and should not be used in newly-written code.
See g_strncasecmp() for a discussion of why this function is deprecated and how to replace it.
```

```
The problem with g_strncasecmp() is that it does the comparison by calling toupper()/tolower(). These functions are locale-specific and operate on single bytes...
````